packages/grafana-data/src/field/fieldDisplay.test.ts

```diff
@@ -64,33 +64,36 @@ describe('FieldDisplay', () => {
 
   it('should restore -Infinity value for base threshold', () => {
     const field = {
-      thresholds: [
-        ({
-          color: '#73BF69',
-          value: null,
-        } as unknown) as Threshold,
-        {
-          color: '#F2495C',
-          value: 50,
-        },
-      ],
+      scale: {
+        mode: ScaleMode.absolute,
+        thresholds: [
+          ({
+            color: '#73BF69',
+            value: null,
+          } as unknown) as Threshold,
+          {
+            color: '#F2495C',
+            value: 50,
+          },
+        ],
+      },
     };
     setFieldConfigDefaults(field);
-    expect(field.thresholds!.length).toEqual(2);
-    expect(field.thresholds![0].value).toBe(-Infinity);
+    expect(field.scale.thresholds!.length).toEqual(2);
```

Comment:
thresholds array is not optional (or nullable) on the Scale type, so not sure ! is needed here